app/src/main/java/blockeq/com/stellarwallet/adapters/TradingPagerAdapter.kt

```diff
@@ -0,0 +1,37 @@
+package blockeq.com.stellarwallet.adapters
+
+import android.support.v4.app.Fragment
+import android.support.v4.app.FragmentManager
+import android.support.v4.app.FragmentPagerAdapter
+import blockeq.com.stellarwallet.fragments.tabs.MyOffersTab
+import blockeq.com.stellarwallet.fragments.tabs.OrderBookTab
+import blockeq.com.stellarwallet.fragments.tabs.TradeTab
+
+class TradingPagerAdapter(fm: FragmentManager) : FragmentPagerAdapter(fm) {
+
+    override fun getItem(position: Int): Fragment {
+        return when (position) {
+            0 -> {
```

Comment:
We could create enum to describe what is 0, 1 etc. This one is a good resource to follow: https://www.journaldev.com/18688/kotlin-enum-class

Example:
`enum class Tabs (var title: String) {
    Trade('Trade'),
    OrderBook('Order Book'),
    Offers('My Offers')
}`

`var x = Tabs.OrderBook
println(x. ordinal) //prints 1
println(x.title) //prints Order Book
println(Tabs.values().size) //prints 3
`